invenio_communities/assets/semantic-ui/js/invenio_communities/members/api/membersApiClient.js

```diff
@@ -0,0 +1,55 @@
+import { http } from "./config";
```

Comment:
I would rethink how we organize the code here, I think we should have a discussion in the team.

Reply:
yes definitely, i just wanted to get a start on it.